### ISSUE TITLE
Behaviour if image is not available

### ISSUE THREAD

**Issue by robations** [timhagn/gbitest#10]
**Question/feature request**

Hi, I have a situation where some elements might not have an image, or the image isn't suitable to use as a background. Current behaviour (tested using the `fluid` property) seems to be to not render an element at all if the image value is null or undefined.

Would it possible to render the GBI element without an image, but still provide the background color as a fallback? This would avoid having to create two very similar document structures for my element.

**Comment by timhagn** [comment]
Hi @robations,

to render the component without an image, you may just hand over an empty Object. But to prevent React from screaming at you in Dev-mode, you have to give `gbi` a dummy fluid object like thus:

```jsx
  const emptyImageData = {
    aspectRatio: 0,
    src: '',
    srcSet: '',
  }
  return (
      <BackgroundImage
        className={className}
        fluid={{}}
        backgroundColor={`#040e18`}
        title="Empty Background"
        id="fullscreenbg"
      >
        {children}
      </BackgroundImage>
  )
```

Have a look at its `index.d.ts` to see which properties are needed for `fixed` respectively : ). Hope it helps!

Best,

Tim.

**Comment by robations** [comment]
Thanks for the extra and detailed info, Tim. I'll test this out and then close, unless you want to do something with this idea in the future.

In the words of an overwrought mother: "Won't somebody think of the `children`?"

**Comment by timhagn** [comment]
Hi @robations,

just released `gbi` `v1.4.0` adding the `keepStatic` switch, which prevents a collapse of the container should `fluid` or `fixed` be empty. You may tell the overwrought mother: "The `children` are saved!" ; )

Best,

Tim.
